content/blog/2019-02-18-stats19.md

```diff
@@ -0,0 +1,1154 @@
+[`0.2.0`](https://cran.r-project.org/web/packages/stats19/index.html)
+has just been release on CRAN, which contains a few improvements, some
+of which are used in this blog post (`ask = FALSE` in `get_stats19()`,
+for example, which makes it even quicker to get data with this package).
+
+We started the package in late 2018 following three main motivations:
+
+1.  The release of the 2017 road crash statistics, which showed
+    worsening road safety in some areas, increasing the importance of
+    making the data more accessible.
+2.  The realisation that many researchers were writing add-hoc code to
+    clean the data, with a huge amount of duplicated (wasted) effort and
+    potential for mistakes to lead to mistakes in the labelling of the
+    data (more on that below).
+3.  An understanding of the concept of ‘modularity’ in software design,
+    following the [Unix
+    philosophy](https://en.wikipedia.org/wiki/Unix_philosophy) that
+    programs should ‘do one thing and do it well’. This realisation has
+    led to code inside the rOpenSci-hosted package
+    [`stplanr`](https://github.com/ropensci/stplanr) being split-out
+    into 2 separate packages already:
+    [`cyclestreets`](https://github.com/Robinlovelace/cyclestreets) and
+    [`stats19`](https://github.com/ropensci/stats19).
+
+The main authors are based at the [Institute for Transport Studies
+(ITS)](https://environment.leeds.ac.uk/transport) and the [Leeds
+Institute for Data Analytics (LIDA)](https://lida.leeds.ac.uk/),
+institutions focussed on transport and data-intensive research. We have
+prior experience writing code to work with road crash data: Robin wrote
+code for an academic paper on cycle safety in Yorkshire based on STATS19
+data (Lovelace, Roberts, and Kellar 2016), and put it in the
+[robinlovelace/bikeR](https://github.com/Robinlovelace/bikeR) repo for
+posterity/reproducibility. Package contributor, Dr Malcolm Morgan, wrote
+code processing different STATS19 data for the Cycling Infrastructure
+Prioritisation Toolkit ([CyIPT](https://www.cyipt.bike)) and put it in
+[cyipt/stats19](https://github.com/cyipt/stats19). The large and complex
+STATS19 data from the UK’s Department for Transport, which is open
+access but difficult-to-use, represented a perfect opportunity for us to
+get stuck into a chunky data processing challenge.
+
+We have a wider motivation: we want the roads to be safer. By making
+data on the nature of road crashes in the UK more publicly accessible
+(in a usable format), we hope this package saves lives.
+
+## What is STATS19 anyway?
+
+One of the reviewer comments from the [rOpenSci review process (which
+has 77 comments - lots of knowledge
+shared)](https://github.com/ropensci/software-review/issues/266) alluded
+to the package’s esoteric name:
+
+> I confess I wish the package name was more expressive–stats19 sounds
+> like an introductory statistics class.
+
+We agree\! However, the priority with the package is to remain faithful
```

Comment:
In this and the following paragraphs it will help if you condense the text to get the main ideas across and try to omit extra phrases like "Why STATS19 and not, say, STATS18 or STATS20? We’re not sure, but..." that distract the reader

Reply:
:+1:

Reply:
OK, I think the best way to condense/address the quote issue was to remove the quote. The flow now goes: names comes and name is DfT, refer to review process and then quickly stress our motivation (cost of road safety numbers). 

See if you both like my edition @stefaniebutland and @Robinlovelace. Thank you Stefanie.

Reply:
👍